File: Task/Array-concatenation/Objective-C/array-concatenation-1.m
NSArray *arr1 = [NSArray arrayWithObjects:[NSNumber numberWithInt:1],
                                          [NSNumber numberWithInt:2],
                                          [NSNumber numberWithInt:3], nil];
NSArray *arr2 = [NSArray arrayWithObjects:[NSNumber numberWithInt:4],
                                          [NSNumber numberWithInt:5],
                                          [NSNumber numberWithInt:6], nil];
NSArray *arr3 = [arr1 arrayByAddingObjectsFromArray:arr2];

File: Task/Array-concatenation/Python/array-concatenation-1.py
arr1 = [1, 2, 3]
arr2 = [4, 5, 6]
arr3 = [7, 8, 9]
arr4 = arr1 + arr2
assert arr4 == [1, 2, 3, 4, 5, 6]
arr4.extend(arr3)
assert arr4 == [1, 2, 3, 4, 5, 6, 7, 8, 9]
